src/Lecture2.hs

```diff
@@ -194,7 +212,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] r = r
+merge l [] = l
+merge (x:xs) (y:ys)
+  | x > y     = y : merge (x:xs) ys
+  | otherwise = x : merge xs (y:ys)
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️

Reply:
really good point. thanx